### PR TITLE
Accessibility for portfolio project

### PR DESCRIPTION
**Pull request by Christelle-12**
In this project we :
-Noticed that some changes are required for the webpage to be accessible.
-made those required changes
-checked our HTML / CSS validation then corrected errors and warnings present
 -Those errors and warnings were:

- Page titles - no issue found

- image text alternatives - changed text alternatives for cards 

- Text headings -changed unwanted h1 to h2

-  Color contrast - changed contrast for texts to be accessible

-  Resize - no issue found

-  Interaction - no issue found 

-  Multimedia  - no issue found

-  Basic structure of HTML - removed <a> tag from <button>

- Moving content - no issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <title>MyPortfolio</title>
```
Comment (binhussen) [inline index.html]: - Kindly There is a page title, <title> in the <head> of the page adequately and briefly describes the content of the page. so please use more descriptive name example `Mark's portifolio` ,`Mark full-stack web developer.... to make  your portfolio more accessible 👍👍

File: style.css [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+}
+
+@font-face {
+  font-family: 'Poppins';
+  src: url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap");
+  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+  font-weight: 900;
+  font-style: normal;
+}
+
```
Comment (binhussen) [inline style.css:32]: - Kindly some of your color contrast is failing, check your result [here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-03-08-at-12-29-41-791?failing), so please update your color contrast by using this [website](https://www.aditus.io/button-contrast-checker/) to make your site more accessible 👍👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Christelle-12  and @Ddiiro , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @Christelle-12 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Congratulations!GIF


## Highlights 🎉 💯👌

- No linter errors present
- Good descriptive pull request👍
- Good at following the Gitflow

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**